import.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
```

Comment:
would be great to make the bash script stop if there's an error. I usually put something like this at the top:
```bash
# Exit when any command fails
set -e
```

Reply:
Added, ty